keda/src/components/scaledjobs/List.tsx

```diff
@@ -43,32 +43,9 @@ export function ScaledJobsList() {
           id: 'triggers',
           label: 'Triggers',
           getValue: item =>
-            item.spec.triggers && item.spec.triggers.length > 0 ? item.spec.triggers[0].type : '-',
-        },
-        {
-          id: 'authentications',
-          label: 'Authentications',
-          getValue: null,
-          render: item => {
-            const triggerAuthenticationRefs = item.spec.triggers
-              .map(trigger => trigger.authenticationRef)
-              .filter(Boolean);
-
-            if (!triggerAuthenticationRefs.length) return <span>-</span>;
-
-            return (
-              <Link
-                key={triggerAuthenticationRefs[0].name}
-                routeName={triggerAuthenticationRefs[0].kind ?? TriggerAuthentication.kind}
-                params={{
-                  name: triggerAuthenticationRefs[0].name,
-                  namespace: item.metadata.namespace,
-                }}
-              >
-                {triggerAuthenticationRefs[0].name}
-              </Link>
-            );
-          },
```

Comment:
had to remove this because of insufficient space on UI considering we would have to use `triggerAuthenticationRefs.map(...)` to render each link to a trigger auth

![image](https://github.com/user-attachments/assets/9c9648be-ec77-4ded-a373-630feb03496c)